style.css

```diff
@@ -0,0 +1,218 @@
+body {
+  background-color: whitesmoke;
+  font-family: Arial, Helvetica, sans-serif;
+  color: rgb(92, 92, 92);
+}
+
+#page-container {
+  position: relative;
+}
+
+#header {
+  background-color: rgb(243, 216, 97);
+  position: fixed;
+  top: 0;
+  min-height: 77px;
+  width: 100%;
+  padding: 0 23px;
+  display: flex;
+  align-items: center;
+  z-index: 2;
+}
+
+#header-img {
+  padding: 11px;
+  width: 100px;
+  height: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-left: 27px;
+}
+
+h1 {
+  padding: 11px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+/* navbar contains the mobile styling */
```

Comment:
- [ ] Kindly phrase your comments as more of suggestions instead of orders. For example if you need a student to delete unused code or comments you can say " Please delete unused or commented out code."
- [ ] Please include an explanation for each requested change. For example in this case you could add: "Please delete unused or commented out code. Deleting unused comments from the code makes it cleaner.".